lib/MetaCPAN/Model/Search.pm

```diff
@@ -79,18 +79,15 @@ sub search_web {
     $page_size //= 20;
     $from      //= 0;
 
-    $search_term
-        =~ s{([ + - = > < ! & | ( ) { } \[ \] ^ " ~ * ? \ / ])}{\\$1}x;
+    $search_term =~ s{([+=><!&|\(\)\{\}[\]\^"~*?\\/])}{\\$1}g;
```

Comment:
Good detective work!  🔍

Reply:
I thought making it not match spaces would be easy. In fact, after half an hour I gave up and just zapped the whitespace!